internal/lookout/ui/src/components/NavBar.tsx

```diff
@@ -1,14 +1,54 @@
 import React from "react"
```

Comment:
The content of this file comes from the parent of 3323ad7f74b7e7a6a07c29e2698736f0ec38c93c; the resulting navigation bar looks like this:

![image](https://github.com/armadaproject/armada/assets/41909795/be509ae5-7ca5-4a85-aee1-0bf067ba2013)